src/preconditions/Permissions.ts

```diff
@@ -21,6 +21,47 @@ export class CorePrecondition extends Precondition {
 			? (message.channel as TextChannel | NewsChannel).permissionsFor(message.client.id!)!
 			: this.dmChannelPermissions;
 		const missing = permissions.missing(required);
-		return missing.length === 0 ? this.ok() : this.error({ message: 'I am missing permissions to run this command.', context: { missing } });
+		return missing.length === 0
+			? this.ok()
+			: this.error({
```

Comment:
Does this not have an identifier?

Reply:
The identifier defaults to the preconditions name.